boa_engine/src/value/mod.rs

```diff
@@ -40,8 +40,8 @@ mod r#type;
 
 pub use conversions::*;
 pub use display::ValueDisplay;
-pub use equality::*;
-pub use hash::*;
+// pub use equality::*;
+// pub use hash::*;
```

Comment:
This was causing a build failure for me locally...
```
error: unreachable `pub` item
  --> boa_engine/src/value/mod.rs:43:9
   |
43 | pub use equality::*;
   | ---     ^^^^^^^^
   | |
   | help: consider restricting its visibility: `pub(crate)`
   |
   = help: or consider exporting it for use by other crates
note: the lint level is defined here
  --> boa_engine/src/lib.rs:41:5
   |
41 |     unreachable_pub,
   |     ^^^^^^^^^^^^^^^
```

Didn't mean to commit it, but easy enough to revert.

Reply:
If this didn't break any APIs, I think you can remove this